### PR TITLE
Adjust the top from 0 to -80.

### PR DESCRIPTION
**Pull request by sunshineo**
If I'm not mistaken in https://github.com/kudago/smart-app-banner/issues/43 Then we need this.

### CODE REVIEW

File: smart-app-banner.css [hunk 1]
```diff
@@ -10,7 +10,7 @@
 .smartbanner {
 	position: absolute;
 	left: 0;
-	top: 0;
+	top: -80;
```
Comment (dy) [inline smart-app-banner.css:13]: Unitless values are senseless here

Comment (sunshineo) [inline smart-app-banner.css:13]: Should have px. But how did this work for me??

Comment (dy) [inline smart-app-banner.css:13]: Well apparently bad values are reset to default values, which is `auto` for `top`.

Comment (sunshineo) [inline smart-app-banner.css]: You are right @dfcreative . I tried auto, and it works. I'll create a different merge request.

Comment (sunshineo) [inline smart-app-banner.css:13]: Hi @dfcreative , I tried auto and it did not work. I'm going to create a separate pull request with 80px.

**Pull request closed without merging**